scene/gui/text_edit.cpp

```diff
@@ -827,15 +827,15 @@ void TextEdit::_notification(int p_what) {
 
 						if (caret_line_wrap_index_map.has(minimap_line) && caret_line_wrap_index_map[minimap_line].has(line_wrap_index) && highlight_current_line) {
 							if (rtl) {
-								RenderingServer::get_singleton()->canvas_item_add_rect(ci, Rect2(size.width - (xmargin_end + 2) - minimap_width, i * 3, minimap_width, 2), theme_cache.current_line_color);
+								RenderingServer::get_singleton()->canvas_item_add_rect(ci, Rect2(size.width - (xmargin_end + 2) - minimap_width, (i * minimap_line_height), minimap_width, minimap_line_height / 2), theme_cache.current_line_color);
```

Comment:
Use `minimap_char_size.y` instead of `minimap_line_height / 2`.

Reply:
Thank You! However whats the likelihood of this feature being merged? its been open for so long :(

Reply:
Maybe because you didn't open a feature request, also the `TextEdit` still has a lot of issues that should be fixed before adding any new feature.

Reply:
Hmmm alright then I see... Thanks